render-test/runner.cpp

```diff
@@ -683,7 +683,8 @@ TestRunner::Impl::Impl(const TestMetadata& metadata, const mbgl::ResourceOptions
               .withMapMode(metadata.mapMode)
               .withSize(metadata.size)
               .withPixelRatio(metadata.pixelRatio)
-              .withCrossSourceCollisions(metadata.crossSourceCollisions),
+              .withCrossSourceCollisions(metadata.crossSourceCollisions)
+              .withKeepRenderData(metadata.mapMode != MapMode::Tile),
```

Comment:
`metadata.mapMode == MapMode::Continuous`? This is useful for static mode too no?

Reply:
some render tests operations rely on cache being preserved, e.g. `wait` operation